### DISCUSSION TITLE
Using a metal case with PedalinoMini?

### DISCUSSION THREAD

**Discussion by MiqViq** [alf45tar/PedalinoMini#227] [Q&A]
Hi, I am planning to build PedalinoMini inside a metal case, will there be any possible issues with that?

I think I am gonna use some sort of rugged sheet metal (1.5-2.5mm thick) and bend it in a shape that is a wide "U" shape on the side. Side panels would probably be wooden pieces. Bottom may be either wood or sheet metal.
I have a friend that has access to CNC routing devices, laser cutters and stuff so the case is gonna have a more professional outlook...

Will the metal case block WiFi or Bluetooth signals as my current ESP 32-board has no external antenna sockets?

**Comment by alf45tar** [comment]
I do not think metal will block wifi/bleutooth signal too much. A concrete wall blocks more.
I suggest to test yourself putting the board inside a metal can.

**Reply by jisv48** [reply]
Do you have a preferred 3D printing company that we can use to print the pedaling case?  Thanks

**Reply by alf45tar** [reply]
I own a 3D printer

**Comment by jisv48** [comment]
Great, build one for me and let me know the cost. Thanks

**Reply by alf45tar** [reply]
Sorry bu I am not providing any printing service yet. If there are more people interested to purchase the PedalinoMini Case 1 I will evaluate a collettive solution.

**Comment by jisv48** [comment]
Awesome,  Thank you

**Comment by potul** [comment]
I have built mine inside an aluminium box, and guess what? Wifi is not reaching... So definitively, metal enclosure has an impact into Wifi reception.
I plan to add an external antenna I have ordered. Once I finish the build I will post a follow up here.
In the meantime I need to open the box if I want to configure it.

**Comment by upgrader-dev** [comment]
Just to share my experience. i'm building a pedalino mini inside a regular 4 footswitches metal pedal. No issue with wifi or BLE :)
